__tests__/Formsy-spec.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { mount } from 'enzyme';
 import sinon from 'sinon';
 
-import Formsy, { addValidationRule } from './..';
+import Formsy, { addValidationRule } from '../src';
```

Comment:
@ThibautMarechal This is the change that makes the tests used the source files not the build files. I haven't been able to find a way in jest for _forbid_ it from importing from the root.